jms/src/main/scala/akka/stream/alpakka/jms/javadsl/JmsSource.scala

```diff
@@ -6,8 +6,8 @@ package akka.stream.alpakka.jms.javadsl
 
```

Comment:
I could have created a separate entry point with all same signatures, like `JmsSource`, `JmsAckSource`, and `JmsTxSource` with the same signatures inside. Just wanted to wait for some opinions from the code review before making these changes.